### PR TITLE
Release v2.0.6

### PR DESCRIPTION
**Pull request by daira**
Test plan (ECC internal): https://docs.google.com/document/d/1EVDxdFRgQR5xBzA2wRsw02sb-yF2h26FyVnSjqfpayg/edit#

### CODE REVIEW

**Comment by mdr0id** [comment]
Minor nit:

Aren't we missing updating chainwork?

See: https://github.com/zcash/zcash/pull/3833/commits/fe0507761a5bd1c35dbc657c6f9ed7f553e55662

**Comment by daira** [comment]
I didn't think we did that on every release?

**Comment by daira** [comment]
`minchainwork` is only there to prevent an attack where the adversary controls the peers of a node when it starts up, and provides it with a completely fake block chain starting from a low difficulty. The idea is to make this infeasible because the node requires a minimum amount of work on the chain. It *cannot* prevent attacks that try to provide a fake chain starting from a high difficulty.

File: doc/authors.md [hunk 1]
```diff
@@ -31,42 +32,42 @@ Marius Kjærstad (6)
...
-Pavel Janík (5)
+Peter Todd (5)
 Karl-Johan Alm (5)
 Johnathan Corgan (5)
-Charlie O'Keefe (5)
 Alex Morcos (5)
 WO (4)
 Philip Kaufmann (4)
-Peter Todd (4)
+Pavel Janík (4)
```
Comment (Eirik0) [inline doc/authors.md]: -Pavel Janík (5)
-Pavel Janik (2)
+Pavel Janík (4)

I am not sure what to make of these numbers.

File: doc/release-notes/release-notes-2.0.6.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      ld --version doesn't work on macOS.
+      Tweak author aliases.
+      Add coding declaration to zcutil/release-notes.py
+      make-release.py: Versioning changes for 2.0.6.
+      make-release.py: Updated manpages for 2.0.6.
+
+Daniel Kraft (1):
+      Add some const declarations where they are appropriate.
+
+Eirik Ogilvie-Wigley (1):
```
Comment (Eirik0) [inline doc/release-notes/release-notes-2.0.6.md]: I had to change somethings around on my computer. I wonder if I have done something wonky that makes me show up twice in this list or if it something to do with how the script works.

File: doc/authors.md (hunk 1 above)
Comment (daira) [reply doc/authors.md]: I was confused by that too. I will investigate it before the next release.

File: doc/release-notes/release-notes-2.0.6.md (hunk 2 above)
Comment (daira) [reply doc/release-notes/release-notes-2.0.6.md]: I suspect a bug in the script. I'll investigate that too, but it does not block the release.

**Comment by daira** [comment]
@zkbot r+

**Comment by daira** [comment]
```
=== Running testscript sprout_sapling_migration.py ===
  File "/home/zcbbworker/latent-0/debian8/build/qa/rpc-tests/test_framework/test_framework.py", line 121, in main
    self.run_test()
  File "/home/zcbbworker/latent-0/debian8/build/qa/rpc-tests/sprout_sapling_migration.py", line 188, in run_test
    self.run_migration_test(self.nodes[1], sproutAddr1, saplingAddr1, 1000)
  File "/home/zcbbworker/latent-0/debian8/build/qa/rpc-tests/sprout_sapling_migration.py", line 138, in run_migration_test
    check_migration_status(node, True, saplingAddr, True, True, False, 0, 1)
  File "/home/zcbbworker/latent-0/debian8/build/qa/rpc-tests/sprout_sapling_migration.py", line 31, in check_migration_status
    assert_equal(non_zero_unmigrated_amount, Decimal(status['unmigrated_amount']) > Decimal('0.00'))
  File "/home/zcbbworker/latent-0/debian8/build/qa/rpc-tests/test_framework/util.py", line 362, in assert_equal
    raise AssertionError("(left == right)%s\n  left: <%s>\n right: <%s>" % (message, str(expected), str(actual)))
Initializing test directory /tmp/testA3zZLx
Running test using '-migrationdestaddress'...
Mining blocks...
waiting for async operation opid-134a10f5-472e-4e20-9aed-b9ec8d412048
Mining to block 494 % 500...
migration operation: [{u'status': u'executing', u'creation_time': 1560966836, u'id': u'opid-6e1a3d56-9409-48e9-bb72-be337bdf8cb6', u'target_height': 500, u'method': u'saplingmigration'}]
waiting for async operation opid-6e1a3d56-9409-48e9-bb72-be337bdf8cb6
result: {u'status': u'success', u'execution_secs': Decimal('15.705304089'), u'creation_time': 1560966836, u'id': u'opid-6e1a3d56-9409-48e9-bb72-be337bdf8cb6', u'result': {u'amount_migrated': u'9.3999', u'migration_txids': [u'3348e6905deeb2e3ae88486aa5bf57d6124ca1cf6af8d0b77fd425367f13f8c3'], u'num_tx_created': 1}, u'target_height': 500, u'method': u'saplingmigration'}
mempool: [u'3348e6905deeb2e3ae88486aa5bf57d6124ca1cf6af8d0b77fd425367f13f8c3']
status: {u'migration_txids': [u'3348e6905deeb2e3ae88486aa5bf57d6124ca1cf6af8d0b77fd425367f13f8c3'], u'finalized_migrated_amount': u'0.00', u'unfinalized_migrated_amount': u'9.3999', u'enabled': True, u'unmigrated_amount': u'0.60', u'finalized_migration_transactions': 0, u'destination_address': u'zregtestsapling1ssqj3f3majnl270985gqcdqedd9t4nlttjqskccwevj2v20sc25deqspv3masufnwcdy67cydyy'}
sprout balance: 0.60000000, sapling balance: 9.39990000
Running test using default Sapling address...
Mining blocks...
waiting for async operation opid-36f4bfc7-822d-44aa-9641-7d0cd26aad31
Mining to block 494 % 500...
migration operation: [{u'status': u'executing', u'creation_time': 1560966889, u'id': u'opid-c70299a5-b9c3-4a0b-8e20-b9308def9b0d', u'target_height': 1000, u'method': u'saplingmigration'}]
waiting for async operation opid-c70299a5-b9c3-4a0b-8e20-b9308def9b0d
result: {u'status': u'success', u'execution_secs': Decimal('15.668860848'), u'creation_time': 1560966889, u'id': u'opid-c70299a5-b9c3-4a0b-8e20-b9308def9b0d', u'result': {u'amount_migrated': u'9.9999', u'migration_txids': [u'8def08d8d59b9b798ac16a5a11f3ca441970b660fa7071548d486ad289aa7176'], u'num_tx_created': 1}, u'target_height': 1000, u'method': u'saplingmigration'}
mempool: [u'8def08d8d59b9b798ac16a5a11f3ca441970b660fa7071548d486ad289aa7176']
status: {u'migration_txids': [u'8def08d8d59b9b798ac16a5a11f3ca441970b660fa7071548d486ad289aa7176'], u'finalized_migrated_amount': u'0.00', u'unfinalized_migrated_amount': u'9.9999', u'enabled': True, u'unmigrated_amount': u'0.00', u'finalized_migration_transactions': 0, u'destination_address': u'zregtestsapling1mvacs4u6vkxu3df8mxn8udc30kpd52ps5v2a4fjk43cyndckxf4tm7ha0fftqztkscv3v4lvlx9'}
sprout balance: 0E-8, sapling balance: 9.99990000
Assertion failed: (left == right)
  left: <True>
 right: <False>
Stopping nodes
Cleaning up
Failed
!!! FAIL: sprout_sapling_migration.py !!!
```

This can't be related specifically to the release PR, and it doesn't block the release. I'll file a ticket for it.

**Comment by daira** [comment]
@zkbot r+

**Comment by daira** [comment]
Filed #4059.

**Comment by daira** [comment]
@Tomas-M pushing the release tag was delayed because of Zcon1. It's there now: https://github.com/zcash/zcash/releases/tag/v2.0.6

**Pull request merged**